content/news/media/media.collection.en.yaml

```diff
@@ -1,3 +1,38 @@
+-
+  date: 2020-09-10
+  link: https://storj.io/blog/2020/09/using-tardigrade-and-the-decentralized-cloud-to-speed-up-sync-times-for-geth/?utm_source=etc-labs&utm_medium=twitter
```

Comment:
Can we remove the `?utm_source=etc-labs&utm_medium=twitter` part of the URL on this?

Reply:
yes